home/ivp-planning/user-interface/the-top-bar.md

```diff
@@ -49,11 +59,6 @@ The text menu contains four submenus:
 |                                                                                |                                                                          |
 | ------------------------------------------------------------------------------ | ------------------------------------------------------------------------ |
 | ![](../../../.gitbook/assets/iVP\_menu\_bar\_layouts.jpg)                      | Opens a small submenu that lets you choose a [window layout](layouts.md) |
-| ![](../../../.gitbook/assets/iVP\_menu\_bar\_settings.jpg)                     | Opens the [settings panel](settings-panel.md)                            |
-| ![](../../../.gitbook/assets/iVP\_menu\_bar\_path\_tool.jpg)                   | Opens the [path panel](path-panel.md)                                    |
-| ![](../../../.gitbook/assets/iVP\_menu\_bar\_render\_camera\_view.jpg)         | Opens the [virtual camera panel](virtual-camera-panel.md)                |
-| ![](../../../.gitbook/assets/iVP\_menu\_bar\_video\_export\_and\_timeline.jpg) | Opens the [camera list panel](camera-list-panel.md)                      |
-| ![](../../../.gitbook/assets/iVP\_menu\_bar\_more\_information.jpg)            | Opens the [iVP-homepage](https://de.i-vp.dev)                            |
-| ![](../../../.gitbook/assets/iVP\_menu\_bar\_help\_faq\_documentation.jpg)     | Opens the [main page](../../) of this documentation                      |
 | ![](../../../.gitbook/assets/iVP\_menu\_error\_reporting.jpg)                  | Opens the [bug report panel](../bug-reporting.md)                        |
-| ![](../../../.gitbook/assets/iVP\_menu\_about\_ivp.jpg)                        | Opens the [changelog panel](changelog-panel.md)                          |
+| ![](../../../.gitbook/assets/iVP\_menu\_bar\_help\_faq\_documentation.jpg)     | Opens the [main page](../../) of this documentation                      |
+| ![](../../../.gitbook/assets/iVP\_menu\_bar\_more\_information.jpg)            | Opens the about panel, containing the [changelog](/changelog-panel.md) and the [3rd Party Licenses](../open-source-licensing.md)                        |
```

Comment:
This is now a dialog and not a panel anymore. Could you update the wording accordingly?